src/Compilers/CSharp/Portable/Symbols/SymbolWithAnnotations.cs

```diff
@@ -1003,6 +1004,13 @@ protected override bool TypeSymbolEquals(TypeSymbolWithAnnotations other, TypeCo
 
                 return base.TypeSymbolEquals(other, comparison);
             }
+
+            public override string ToDisplayString(SymbolDisplayFormat format)
+            {
+                var underlyingType = _underlying.TypeSymbol;
+                var str = underlyingType.ToDisplayString(format);
+                return underlyingType.IsValueType ? str : str + "?";
```

Comment:
>return underlyingType.IsValueType ? str : str + "?"; [](start = 16, length = 52)

Is there a test for this code path? We probably should use ```TypeSymbol``` property and add "?" only if it is not ```Nullable<T>```. Also, is addition of the "?" supposed to be driven by the format?

Reply:
Will follow up in a subsequent PR.